keep-ui/app/workflows/[workflow_id]/executions.tsx

```diff
@@ -85,14 +88,7 @@ interface Pagination {
   offset: number;
 }
 
-export default function WorkflowDetailPage({
-  params,
-}: {
-  params: { workflow_id: string };
-}) {
-  const router = useRouter();
-  const { data: session, status, update } = useSession();
-
+const OverViewContent = ({ workflow_id }: { workflow_id: string }) => {
```

Comment:
Let's rename this component and move it to its own file, `keep-ui/app/workflows/[workflow_id]/ui/WorkflowOverview.tsx
```suggestion
const WorkflowOverview = ({ workflow_id }: { workflow_id: string }) => {
```
`